packages/dnb-eufemia/src/extensions/forms/Field/Password/Password.tsx

```diff
@@ -0,0 +1,185 @@
+/**
+ * Web Input Component
+ *
+ */
+
+import React, {
+  useContext,
+  useRef,
+  useState,
+  ElementRef,
+  MutableRefObject,
+} from 'react'
+import classnames from 'classnames'
+import SharedContext from '../../../../shared/Context'
+import StringField, { Props as StringFieldProps } from '../String'
+
+import { InputProps, SubmitButton } from '../../../../components/Input'
+import { dispatchCustomElementEvent } from '../../../../shared/component-helper'
+import IconView from '../../../../icons/view'
+import IconViewOff from '../../../../icons/hide'
+import IconViewMedium from '../../../../icons/view_medium'
+import IconViewOffMedium from '../../../../icons/hide_medium'
+import useErrorMessage from '../../hooks/useErrorMessage'
+import useDataValue from '../../hooks/useDataValue'
+
+export type PasswordProps = Omit<StringFieldProps, 'innerRef'> & {
+  /**
+   * Fires when the input toggles to show the password.
+   */
+  onShowPassword?: (event: React.MouseEvent<HTMLButtonElement>) => void
+  /**
+   * Fires when the input toggles to hide the password.
+   */
+  onHidePassword?: (event: React.MouseEvent<HTMLButtonElement>) => void
+  /**
+   * The sizes you can choose is small (1.5rem), default (2rem), medium (2.5rem) and large (3rem) are supported component sizes. Defaults to default / null. Also, if you define a number like size="2" then it will be forwarded as the input element attribute.
+   */
+  size?: InputProps['size']
+  /**
+   * Sets the language used for aria-label translations. Defaults to `Provider` `locale`.
+   */
+  lang?: InputProps['lang']
+  /**
+   * ElememntRef passed on to the password input element.
+   */
+  innerRef?: MutableRefObject<HTMLInputElement>
+  /**
+   * @deprecated in v11, use use `locales`prop on `Provider` and override `passwordShowPasswordLabel` instead.
+   */
+  show_password?: string
+  /**
+   * @deprecated in v11, use use `locales`prop on `Provider` and override `passwordHidePasswordLabel` instead.
+   */
+  hide_password?: string
+  /**
+   * @deprecated in v11, use `onShowPassword` instead.
+   */
+  on_show_password?: (event: React.MouseEvent<HTMLButtonElement>) => void
+  /**
+   * @deprecated in v11, use `onHidePassword` instead.
+   */
+  on_hide_password?: (event: React.MouseEvent<HTMLButtonElement>) => void
+}
+
+function Password(props: PasswordProps) {
+  const [hidden, setHidden] = useState<boolean>(true)
+
+  const sharedContext = useContext(SharedContext)
+  const translations = sharedContext.getTranslation(props).Forms
+
+  const errorMessages = useErrorMessage(props.path, props.errorMessages, {
+    required: translations.passwordRequired,
+    pattern: translations.passwordErrorPattern,
```

Comment:
```suggestion
    required: translations.inputErrorRequired,
    pattern: translations.inputErrorPattern,
```

Both are so edge cases, that I think we rather should use the generalized once.